starter-code/src/ContactsList.jsx

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from "react";
+import contacts from './contacts.json';
+
+
+const displayedContacts = [];
+for (let i = 0; i < 5; i++) {
+  displayedContacts.push(contacts[i])
+}
+
+
+
+
+export default class contactsList extends Component {
```

Comment:
You should start your classes / components with a capital letter : )    >> ContactList
It's a convention.